camel-route-support/src/main/java/io/kaoto/backend/metadata/parser/step/camelroute/CamelRestDSLParseCatalog.java

```diff
@@ -70,6 +74,55 @@ private static Step getVerbStep(String verb) {
         return step;
     }
 
+    @NotNull
```

Comment:
Seriously? You are moved down a couple of lines because of new code and suddenly you are all brand new?